src/index.html

```diff
@@ -32,15 +32,15 @@
     content="https://github.com/chryz-hub/chryz-hub.github.io/blob/master/public/myocatact1%20(1).svg" />
 
   <!-- FAVICON -->
-  <link rel="icon" type="image/ico" href="./src/assets/images/favicon.ico" />
+  <link rel="icon" type="image/ico" href="./assets/favicon.ico" />
 
   <title>Chryz-hub Community</title>
 
   <!-- STYLES -->
   <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet"
     integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
-  <link rel="stylesheet" href="./css/styles.css" />
-  <link rel="stylesheet" href="./css/global.css" />
+  <link rel="stylesheet" href="/src/css/global.css" />
```

Comment:
Think this messed up little bit of the styling take a look at the `src` is not needed